questions.md

```diff
@@ -0,0 +1,5 @@
+# Questions on pull requests
+- What is the definition of the base branch
+- What is the definition of the head branch
```

Comment:
After initializing a pull request, you'll see a review page that shows a high-level overview of the changes between your branch (the compare branch) and the repository's base branch. https://help.github.com/articles/about-pull-requests/